server/sonar-web/src/main/js/apps/issues/sidebar/TypeFacet.js

```diff
@@ -61,12 +63,12 @@ export default class TypeFacet extends React.PureComponent {
     this.props.onChange({ [this.property]: [] });
   };
 
-  getStat(type: string): ?number {
+  getStat(type /*: string */) /*: ?number */ {
     const { stats } = this.props;
     return stats ? stats[type] : null;
   }
 
-  renderItem = (type: string) => {
+  renderItem = (type /*: string */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)